tests/conftest.py

```diff
@@ -15,11 +16,16 @@ def ldap(variables):
 
 
 @pytest.fixture
-def passcode(secret_seed):
-    totp = pyotp.TOTP(secret_seed)
-    return totp.now()
+def passcode(secret_seed, counter):
+    hotp = pyotp.HOTP(secret_seed)
+    return hotp.at(counter)
 
 
 @pytest.fixture
 def urls(variables):
     return variables['RPs_urls']
+
+
+def counter():
+    r = requests.get("http://api.numeri.xyz/v1/9162ce23-2206-44f5-a43d-eb046bcd0690/increase")
```

Comment:
Let's move the counter API endpoint to a config entry. This way we cannot distinct different environments and the counters they use.